### ISSUE TITLE
MS4: Incorporate Peer Review Feedback

### ISSUE THREAD

**Issue by Farhan-Faisal** [UBC-MDS/DSCI522-2425-22-wine-chromatic-profile#53]
Review Comments:

**Docker:**

- [x] 1. the URL takes you to a screen with just a 'data' folder. You have to click on it to see the files. I tested this with Tiffany and found this small code fix in the docker-compose.yml file will allow the URL to take straight to all the files:

volumes: - "${PWD}:/app/data" working_dir: /app/data
'volumes' and 'working_dir' should match, that will solve the issue.

**Results:**

- [ ] While the report mentions multicollinearity checks and feature distributions, it retains all features, including those with minimal distinction between classes (ex: residual sugar). This can be further analyzed to explore feature selection methods for potential improvements.

- [ ] The exceptionally high test scores (accuracy of 0.991, precision of 0.978, recall of 0.987) raise concerns about overfitting, but there isn't enough evidence to justify or refute this concern. Something like a comparison with validation set performance or learning curves could be useful for this.

- [x] A little more flushed-out explanation of the figures like explaining trends in feature distributions or insights from the PR curve could be useful for readers.

_____

- [ ] The importance of the research question is not clearly articulated. The Introduction section would benefit from a more thorough exploration of prior research or recent advancements in the topic of interest. Additionally, providing a clearer background on how this research fits within the broader field would enhance the context and framing.

- [x] The directory contains cache files, such as .ipynb_checkpoints, that are being pushed to GitHub. These files should be added to the .gitignore file to prevent them from being included in the repository.

- [x] While having a LICENSE.md file in the repository is sufficient to define usage terms, including a License section in the README increases visibility, saves users time by summarizing the license or linking to it directly, and follows best practices for open-source projects. It ensures that users quickly understand the licensing terms without searching for another file and adds a polished, professional touch to the repository. Additionally, some organizations recommend including license details in the README for clarity and compliance with open-source guidelines.

_____

Improvements:

- [x] I understand that the document aims to demonstrate the use of documentation techniques; however, the introduction and conclusion lack details regarding the potential use cases of the data and the broader impact of the model. 
- [x] My key takeaway was that the model performed well, but the document could benefit from a few lines addressing the problem, the proposed solution, and the potential impact of the model. 
- [ ] Additionally, a comparison with other models applied to the same dataset would add valuable context. Just to make the report more cohesive.

_____
